armotypes/portaltypes.go

```diff
@@ -53,124 +51,6 @@ type PortalBase struct {
-	AttributeTag           = "tag"
-	AttributeCluster       = "cluster"
-	AttributeNamespace     = "namespace"
-	AttributeKind          = "kind"
-	AttributeName          = "name"
-	AttributeContainerName = "containerName"
-	AttributeApiVersion    = "apiVersion"
-	AttributeWorkloadHash  = "workloadHash"
-	AttributeIsIncomplete  = "isIncomplete"
-	AttributeSensor        = "sensor"
-	AttributePath          = "path"
-	AttributeResourceID    = "resourceID"
-	AttributeContainerScanId = "containerScanId"
-)
-
-// Repository scan related attributes
-const (
-	AttributeRepoName      = "repoName"
-	AttributeRepoOwner     = "repoOwner"
-	AttributeRepoHash      = "repoHash"
-	AttributeBranchName    = "branch"
-	AttributeDefaultBranch = "defaultBranch"
-	AttributeProvider      = "provider"
-	AttributeRemoteURL     = "remoteURL"
-
-	AttributeLastCommitHash     = "lastCommitHash"
-	AttributeLastCommitterName  = "lastCommitterName"
-	AttributeLastCommitterEmail = "lastCommitterEmail"
-	AttributeLastCommitTime     = "lastCommitTime"
-
-	AttributeFilePath          = "filePath"
-	AttributeFileType          = "fileType"
-	AttributeFileDir           = "fileDirectory"
-	AttributeFileUrl           = "fileUrl"
-	AttributeFileHelmChartName = "fileHelmChartName"
-
-	AttributeLastFileCommitHash     = "lastFileCommitHash"
-	AttributeLastFileCommitterName  = "lastFileCommitterName"
-	AttributeLastFileCommitterEmail = "LastFileCommitterEmail"
-	AttributeLastFileCommitTime     = "lastFileCommitTime"
-
-	AttributeUseHTTP       = "useHTTP"
-	AttributeSkipTLSVerify = "skipTLSVerify"
-)
-
-// rego-library attributes
-const (
-	AttributeImageScanRelated     = "imageScanRelated"
-	AttributeImageRelatedControls = "imageRelatedControls"
-	AttributeHostSensorRule       = "hostSensorRule"
-	AttributeHostSensor           = "hostSensor"
-)
-
-// PortalDesignator represents a single designation option
-type PortalDesignator struct {
-	DesignatorType DesignatorType `json:"designatorType" bson:"designatorType"`
-	// A specific Workload ID
-	WLID string `json:"wlid,omitempty" bson:"wlid,omitempty"`
-	// An expression that describes applicable workload IDs
-	WildWLID string `json:"wildwlid,omitempty" bson:"wildwlid,omitempty"`
-	// A specific Secret ID
-	SID string `json:"sid,omitempty" bson:"sid,omitempty"`
-	// Attributes that describe the targets
-	Attributes map[string]string `json:"attributes" bson:"attributes"`
-}
-
-// Worker nodes attribute related consts
-const (
-	AttributeWorkerNodes             = "workerNodes"
-	WorkerNodesmax                   = "max"
-	WorkerNodeslastReported          = "lastReported"
-	WorkerNodeslastReportDate        = "lastReportDate"
-	WorkerNodesmaxPerMonth           = "maxPerMonth"
-	WorkerNodesmaxReportGUID         = "maxReportGUID"
-	WorkerNodesmaxPerMonthReportGUID = "maxPerMonthReportGUID"
-	WorkerNodeslastReportGUID        = "lastReportGUID"
-)
-
 // PortalCluster holds cluster data from portal BE
 type PortalCluster struct {
```

Comment:
Shouldn't this move to the config service package?

Reply:
I moved only the customer struct since it's the only one which sufer from circular dependcies